mitodl/open-discussions
# Add downvoting to comments REST API

#### What are the relevant tickets?
Part of #84 

#### What's this PR do?
Adds downvoting for comments

#### How should this be manually tested?
Go to `/api/v0/comment/{comment_id_here}/`, type 'True' for downvoted and clear upvoted, then submit the form. The API page should refresh and the new detail view should show that the comment is now downvoted.
